docs/management/cases/setup-cases.asciidoc

```diff
@@ -35,6 +35,11 @@ least once, which creates a user profile.
 | Give access to view and delete cases | `Read` for the *Cases* feature under
 *Management* with the deletion sub-feature enabled.
 
+| Give access to add alerts to cases
```

Comment:
At the moment you can't attach alerts to Management Cases. Only Observability or Security Solution Cases. So I think we can omit this?

Reply:
You're right!  I'll cancel this PR.